packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx

```diff
@@ -141,6 +141,7 @@ const SidebarNavigation: React.FC<SidebarNavigationProps> = () => {
           <SiteNavNestList isOpen={gettingStartedOpen}>
             <SiteNavItem>
               <SiteNavAnchor to="/getting-started/engineering">Engineering Guidelines</SiteNavAnchor>
+              <SiteNavAnchor to="/getting-started/design">Design Guidelines</SiteNavAnchor>
```

Comment:
Gunna ask you to clean up some technical debt here that we've missed. Each of these Nav Anchors need to be wrapped in a NavItem.

```
            <SiteNavItem>
              <SiteNavAnchor to="/getting-started/engineering">Engineering Guidelines</SiteNavAnchor>
            </SiteNavItem>
            <SiteNavItem>
              <SiteNavAnchor to="/getting-started/design">Design Guidelines</SiteNavAnchor>
            </SiteNavItem>
            <SiteNavItem>
              <SiteNavAnchor to="/getting-started/working-guide">How to work with us</SiteNavAnchor>
            </SiteNavItem>
            <SiteNavItem>
              <SiteNavAnchor to="/getting-started/about-paste">About Paste</SiteNavAnchor>
            </SiteNavItem>
```

Reply:
Can do!

Reply:
@SiTaggart just pushed up this change - lmk if that all looks right

Reply:
Looks great